tests/installation/partitioning_firstdisk.pm

```diff
@@ -22,7 +22,18 @@ sub take_first_disk_storage_ng {
     return unless is_storage_ng;
     send_key $cmd{guidedsetup};    # select guided setup
     assert_screen 'select-hard-disks';
-    assert_and_click 'hard-disk-dev-sdb-selected';    # Unselect second drive
+    if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
+        assert_screen 'hard-disk-dev-sdb-selected';
+        if (match_has_tag 'hotkey_d') {
+            send_key 'alt-d';
+        }
+        elsif (match_has_tag 'hotkey_e') {
+            send_key 'alt-e';
+        }
+    }
+    else {
+        assert_and_click 'hard-disk-dev-sdb-selected';    # Unselect second drive
```

Comment:
Hm, I am thinking: It could potentially be easier to use a hotkey after all in all modes. Or the following:

```perl
assert_screen 'hard-disk-dev-sdb-selected';
if (get_var('VIDEOMODE', '') =~ /text/ssh-x/) {
    send_key $cmd{select_second_disk};
}
else {
    assert_and_click 'hard-disk-dev-sdb-selected';
}
```

Reply:
But also need the same condition checking to define the $cmd{select_second_disk} according to needle tag.